plugins/basic/python/tk_3dsmaxplus_basic/plugin_bootstrap.py

```diff
@@ -88,7 +88,7 @@ def bootstrap_toolkit(root_path):
 
     # start engine
     sgtk_logger.info("Starting the 3dsmaxplus engine.")
-    toolkit_mgr.bootstrap_engine("tk-3dsmaxplus", entity)
+    toolkit_mgr.bootstrap_engine_async("tk-3dsmaxplus", entity)
```

Comment:
Just for completeness, can we also add a failed callback (see [docs](http://developer.shotgunsoftware.com/tk-core/bootstrap.html#sgtk.bootstrap.ToolkitManager.bootstrap_engine_async))? 

```
def failed_callback(phase, exception):
    '''
    Called by the asynchronous bootstrap if an exception is raised.

    :param phase: Indicates in which phase of the bootstrap the exception
                  was raised. An integer constant which is either
                  ToolkitManager.TOOLKIT_BOOTSTRAP_PHASE or
                  ToolkitManager.ENGINE_STARTUP_PHASE. The former if the
                  failure happened while the system was still bootstrapping
                  and the latter if the system had switched over into the
                  Toolkit startup phase. At this point, the running core API
                  instance may have been swapped over to another version than
                  the one that was originally loaded and may need to be reset
                  in an implementation of this callback.

    :param exception: The python exception that was raised.
    '''
```

Maybe just print the error message to the max console. 

The user story here is that something goes wrong during bootstrap, and as a user i want an message to tell me that shotgun didn't load correctly. If you could run a quick test, simulate a failure perhaps and see that the user experience seems ok on failure, that would be great!